sandbox/seccomp/compiler.go

```diff
@@ -66,18 +70,47 @@ func (c *Compiler) VersionInfo() (string, error) {
 	}
 	raw := bytes.TrimSpace(output)
 	// Example valid output:
-	// 7ac348ac9c934269214b00d1692dfa50d5d4a157 2.3.3 03e996919907bc7163bc83b95bca0ecab31300f20dfa365ea14047c698340e7c
-	// 111 chars + wiggle room
-	if len(raw) > 120 {
-		return "", fmt.Errorf("invalid version-info length: %q", raw)
-	}
+	// 7ac348ac9c934269214b00d1692dfa50d5d4a157 2.3.3 03e996919907bc7163bc83b95bca0ecab31300f20dfa365ea14047c698340e7c bpf-actlog
 	if match := validVersionInfo.Match(raw); !match {
 		return "", fmt.Errorf("invalid format of version-info: %q", raw)
 	}
 
 	return string(raw), nil
 }
 
+// GetLibseccompVersion parses the output of VersionInfo and provides the
+// libseccomp version
+func GetLibseccompVersion(versionInfo string) (string, error) {
```

Comment:
(nitpick) I wonder if something like:
```
type VersionInfo string
func (vi VersionInfo) LibseccompVersion() (string, error)
func (vi VersionInfo) Features() (string, error)
func (vi VersionInfo) HasFeature(feature string) (bool, error)
```
would be more "go-ish". I played a bit with this and here is a full diff with my idea:  http://paste.ubuntu.com/p/WWg6Pm8jRG/